src/main/java/view/RoundView.java

```diff
@@ -9,12 +9,7 @@
 import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
 
-import javax.swing.BorderFactory;
-import javax.swing.BoxLayout;
-import javax.swing.ImageIcon;
-import javax.swing.JButton;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
+import javax.swing.*;
```

Comment:
wait guys I just realised this goes against checkstyle, but also without it I get some import errors?

Reply:
Its fine if it goes against the 207 checkstyle file we have, as long as we can justify why we made this decision. So we can leave it as is 
![image](https://github.com/user-attachments/assets/a0e5ced9-1bb1-4e85-8403-51cebb67d084)